### ISSUE TITLE
Module 3, req.param not recieving parameter

### ISSUE THREAD

**Issue by SanjayHarkaj** [hwz/chirp#20]
In module-3/Finished/routes/api.js
there is 's' missing in param so req.param.id doesnt work replace param with params :)

**Comment by acamogh** [comment]
True, had same problem and solved it. Please find screenshot of working code
![screen shot 2016-04-19 at 3 15 41 pm](https://cloud.githubusercontent.com/assets/7515396/14635026/41080e46-0642-11e6-9b64-1cba318cb980.png)
